LFS171x/docs/discovering-blockchain-technologies.md

```diff
@@ -123,22 +122,22 @@ According to [hyperledger.org](https://www.hyperledger.org),
 
 **Consensus** refers to a system of ensuring that parties agree to a certain state of the system as the true state.
 
-## Video: The Difference between DLT and Blockchains (Brian Behlendorf)
-
-[![The Difference between DLT and Blockchains (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/02IAHzvCh34)
-
 ## Blockchains (Continued)
 
-**Blockchain** is a specific form or subset of distributed ledger technologies, which constructs a chronological chain of blocks, hence the name 'block-chain'. A **block** refers to a set of transactions that are bundled together and added to the chain at the same time. In the Bitcoin blockchain, the miner nodes bundle unconfirmed and valid transactions into a block. Each block contains a given number of transactions. In the Bitcoin network, miners must solve a cryptographic challenge to propose the next block. This process is known as '**proof of work**', and requires significant computing power. We shall discuss proof of work in more detail in the Consensus Algorithms section. For a brief history of blockchain technology, please click here.
+**Blockchain** is a specific form or subset of distributed ledger technologies (DLTs), which constructs a chronological chain of blocks, hence the name 'block-chain'.  Examples of other DLTs are tangle and hashgraph.
```

Comment:
I am not sure I would call tangle and hashgraph DLTs. They are pieces that make up other Distributed Ledgers. Tangle being a storage structure used by IOTA, and hashgraph being a consensus mechanism. We should probably reference the other DLTs that are already covered in this course -- https://github.com/hyperledger/education/blob/import-edx-content/LFS171x/docs/discovering-blockchain-technologies.md#other-open-source-permissioned-distributed-ledgers.

Reply:
OK. I have referenced those ones instead & pushed the commit. Thanks!